etc/spelling.json

```diff
@@ -1,20 +1,149 @@
 {
   "version": "latest",
   "language": "en",
+  "dictionaries": [
+    "bash",
+    "companies",
+    "cpp",
+    "csharp",
+    "css",
+    "en-gb",
+    "en_US",
+    "go",
+    "html",
+    "latex",
+    "misc",
+    "node",
+    "npm",
+    "php",
+    "powershell",
+    "python",
+    "softwareTerms",
+    "typescript"
+  ],
+  "minWordLength": 4,
   "words": [
-    "Seiso",
-    "EPT",
-    "WAPT",
-    "BaU",
-    "ServDev",
-    "FFIEC",
-    "ISO",
-    "HITRUST",
-    "HIPAA",
-    "PCI",
-    "PCI-DSS",
-    "FedRAMP",
-    "SOW",
-    "MSA"
+    "aircrack",
+    "aireplay",
+    "airodump",
+    "axample",
+    "bau",
+    "bigip",
+    "bruteforce",
+    "bruteforcing",
+    "casp",
+    "cissp",
+    "connstring",
+    "cors",
+    "crackmapexec",
+    "csrf",
+    "cvss",
+    "dast",
+    "deauthenticate",
+    "deauthentication",
+    "dehashed",
+    "dialoghandler",
+    "dmarc",
+    "dnsdumpster",
+    "domainkey",
+    "dpapi",
+    "ept",
+    "essid",
+    "essids",
+    "fedramp",
+    "ffiec",
+    "gcux",
+    "gcwn",
+    "giac",
+    "gitleaks",
+    "gpen",
+    "gppa",
+    "gssp",
+    "hashcat",
+    "hcxpcaptool",
+    "hcxtools",
+    "hipaa",
+    "hitrust",
+    "hkey",
+    "hmac",
+    "hotp",
+    "hsts",
+    "icanhazip",
+    "impacket",
+    "infra",
+    "iso",
+    "johntheripper",
+    "kerberoast",
+    "kerberoasting",
+    "llmnr",
+    "lm",
+    "lsass",
+    "meraki",
+    "microsoft",
+    "minidump",
+    "mitm",
+    "msa",
+    "nbns",
+    "netbios",
+    "netlogon",
+    "netntlm",
+    "netntlmv",
```

Comment:
Does this mean it will catch "netntlmv2"?

Reply:
Yes, I'm fairly certain the way `cspell` works would split NetNTLMv2 into "netntlmv" and "2", and then ignore "2" for being a number and < 4 characters.